test/com/rjmetrics/sweet_liberty/integration/t_core.clj

```diff
@@ -518,6 +528,15 @@
                (:headers result) => {"Content-Type" "application/json; charset=UTF-8"
                                      "Vary" "Accept"})))
 
+(facts "about /insert-with-ring-response"
+       (fact "it will return the response that was provided"
+             (let [result (-> (request :post "/insert-with-ring-response")
+                              (body {:state false})
+                              handler)]
+               (json/read-str (slurp (:body result)) :key-fn keyword)
+               => (contains {:errors "from processable?"})
+               (:status result) => 400)))
```

Comment:
It would be better for the status in the test should be something other than `400` because `400` is the value for the standard case.